scripts/local-deployment/localDeploymentLib.ts

```diff
@@ -310,35 +251,12 @@ export const getLocalNetworks = async (
       rollup: data.rollup,
       sequencerInbox: data['sequencer-inbox'],
     },
-    explorerUrl: '',
-    isArbitrum: true,
     isCustom: true,
-    blockTime: 0.25,
     name: 'OrbitLocal',
     retryableLifetimeSeconds: 7 * 24 * 60 * 60,
-    nitroGenesisBlock: 0,
-    nitroGenesisL1Block: 0,
-    depositTimeout: 900000,
-    tokenBridge: {
-      l1CustomGateway: '',
-      l1ERC20Gateway: '',
-      l1GatewayRouter: '',
-      l1MultiCall: '',
-      l1ProxyAdmin: '',
-      l1Weth: '',
-      l1WethGateway: '',
-
-      l2CustomGateway: '',
-      l2ERC20Gateway: '',
-      l2GatewayRouter: '',
-      l2Multicall: '',
-      l2ProxyAdmin: '',
-      l2Weth: '',
-      l2WethGateway: '',
-    },
+    isTestnet: false
```

Comment:
I think it doesn't make any difference atm (judging by the SDK's code), but shouldn't this be true?

Reply:
let's make this true
```suggestion
    isTestnet: true
```